app/assets/stylesheets/application.scss

```diff
@@ -111,7 +112,17 @@ ul .player-panel {
   margin-top: 20px;
 }
 
-ul .player-panel .user-pic :hover {
+.player-panel .user-pic :hover {
```

Comment:
Merge rule `.player-panel .user-pic :hover` with rule on line 103<br>Selector should have depth of applicability no greater than 2, but was 3